### ISSUE TITLE
/api/v1/importer/{name}/report does not match openapi spec

### ISSUE THREAD

**Issue by carlosthe19916** [trustification/trustify#960]
According to the openapi.yaml spec the field `items.report` is a STRING but the actual response of the backend indicate that it is an OBJECT.

```json
{
    "items": [
        {
            "id": "b48319a9-50ca-4607-953f-0f897d03fbec",
            "importer": "osv-pypa",
            "creation": "2024-11-01T12:05:33.198269Z",
            "report": {
                "endDate": "2024-11-01T12:05:33.196614058Z",
                "startDate": "2024-11-01T12:05:32.390363356Z",
                "numberOfItems": 0
            }
        },      
    ],
    "total": 1
}
```

![Screenshot From 2024-11-01 13-31-23](https://github.com/user-attachments/assets/231e6408-28b3-410b-9ca7-69b1003ed05b)

**Comment by ctron** [comment]
I am sorry, but I can't reproduce this issue:

![image](https://github.com/user-attachments/assets/993127c1-207b-4571-9993-cd016afc43ea)

**Comment by carlosthe19916** [comment]
@ctron In your screenshot there is a field `report` whose type is `any`; the `report` schema should be something like 

```json
"report": {
  "endDate": "string",
  "startDate": "string",
  "numberOfItems": "number",
}
```
The field `report` schema should match whatever is the actual response of the backend. `any` means "everything", the "report" can have values from numbers, strings, arrays, objects, array of objects, etc.

**Comment by ctron** [comment]
Ok, should be fixed by #971
